### PR TITLE
config: update the status checks for the pingcap/docs repository 

### PR DESCRIPTION
**Pull request by qiancai**
Related PR: 
master: https://github.com/pingcap/docs/pull/9354
v5.4: https://github.com/pingcap/docs/pull/9349
v6.1: https://github.com/pingcap/docs/pull/9353

### CODE REVIEW

**Review by ran-huang — COMMENTED** [review]
lgtm

**Review by shczhen — APPROVED** [review]

File: prow/config/config.yaml [hunk 1]
```diff
@@ -565,7 +565,8 @@ branch-protection:
               required_status_checks:
                 contexts:
                   - "jenkins-docs/verify"
-                  - "pull"
+                  - "tidb-check"
+                  - "tidb-cloud-check"
```
Comment (Mini256) [inline prow/config/config.yaml:569]: Which PR will change this workflow in `master` branch?

**Comment by Mini256** [comment]
/cc @wuhuizuo 
PTAL~

File: prow/config/config.yaml (hunk 1 above)
Comment (qiancai) [reply prow/config/config.yaml:569]: yes,  https://github.com/pingcap/docs/pull/9354/files

**Review by Mini256 — APPROVED** [review]

**Pull request merged**